From a1b3784e92aec549a8e22c25a193647687bb7a7e Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Mon, 15 Sep 2014 16:13:55 +0800
Subject: [PATCH 0950/5242] MLK-12423: ARM: dts: imx6sl: remove always-on for
 vddpu

commit  c2d21b3c2d4daab1884548bb765be487b46df132 from
https://source.codeaurora.org/external/imx/linux-imx.git

PU can be dynamically turned off or on, so we need remove
"regulator-always-on" property.

Signed-off-by: Robin Gong <b38343@freescale.com>
(cherry picked from commit f3c0df15fbecce36cae531a4a919d544f9ea8e2a)
(cherry picked from commit ec113127f090c225dd731383b6fb950c02ae1f0b)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sl.dtsi |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6sl.dtsi b/arch/arm/boot/dts/imx6sl.dtsi
index 3719861..9b922dfe 100644
--- a/arch/arm/boot/dts/imx6sl.dtsi
+++ b/arch/arm/boot/dts/imx6sl.dtsi
@@ -642,7 +642,8 @@
 					regulator-name = "vddpu";
 					regulator-min-microvolt = <725000>;
 					regulator-max-microvolt = <1450000>;
-					regulator-always-on;
+					regulator-enable-ramp-delay = <150>;
+					regulator-boot-on;
 					anatop-reg-offset = <0x140>;
 					anatop-vol-bit-shift = <9>;
 					anatop-vol-bit-width = <5>;
-- 
1.7.9.5

